_data/simple-icons.json

```
@@ -7584,6 +7584,12 @@
             "source": "https://brand.netflix.com/en/assets/brand-symbol",
             "guidelines": "https://brand.netflix.com/en/assets/brand-symbol"
         },
+        {
+            "title": "Netgate",
+            "hex": "1475CF",
+            "source": "https://www.netgate.com/",
+            "guidelines": "https://www.netgate.com/"
```

```suggestion
            "source": "https://www.netgate.com"
```

Where on this page is the SVG available? I can only see PNGs there currently.